docs/resources/Guild.md

```diff
@@ -854,13 +855,14 @@ Create a new [role](#DOCS_TOPICS_PERMISSIONS/role-object) for the guild. Require
 
 ###### JSON Params
 
-| Field       | Type    | Description                                                    | Default                        |
-| ----------- | ------- | -------------------------------------------------------------- | ------------------------------ |
-| name        | string  | name of the role                                               | "new role"                     |
-| permissions | string  | bitwise value of the enabled/disabled permissions              | @everyone permissions in guild |
-| color       | integer | RGB color value                                                | 0                              |
-| hoist       | boolean | whether the role should be displayed separately in the sidebar | false                          |
-| mentionable | boolean | whether the role should be mentionable                         | false                          |
+| Field       | Type                                      | Description                                                                               | Default                        |
+| ----------- | ----------------------------------------- | ----------------------------------------------------------------------------------------- | ------------------------------ |
+| name        | string                                    | name of the role                                                                          | "new role"                     |
+| permissions | string                                    | bitwise value of the enabled/disabled permissions                                         | @everyone permissions in guild |
+| color       | integer                                   | RGB color value                                                                           | 0                              |
+| hoist       | boolean                                   | whether the role should be displayed separately in the sidebar                            | false                          |
+| icon        | ?[image data](#DOCS_REFERENCE/image-data) | base64 64x64 png/jpeg image for the role icon (if the guild has the `ROLE_ICONS` feature) | null                           |
```

Comment:
```suggestion
| icon        | ?string | the role's [icon hash](#DOCS_REFERENCE/image-formatting) (if the guild has the `ROLE_ICONS` feature) | null                           |
```
The example you have given is a hash and there's also a CDN URL for role icons, hence it would make more sense that this is the role's icon hash rather than Base64 encoded image data.